src/app-layout/__tests__/drawers.test.tsx

```diff
@@ -33,11 +33,18 @@ describeEachAppLayout(() => {
     };
     const { wrapper } = renderComponent(<AppLayout contentType="form" {...emptyDrawerItems} />);
 
-    expect(wrapper.findDrawersTriggers()!).toHaveLength(0);
+    expect(wrapper.findDrawersTriggers()).toHaveLength(0);
+    expect(wrapper.findToolsToggle()).toBeTruthy();
   });
 
-  test('renderds drawers with the tools', () => {
-    const { wrapper } = renderComponent(<AppLayout tools="Test" {...singleDrawer} />);
+  test('should apply drawers treatment to the tools if at least one other drawer is provided', () => {
+    const { wrapper } = renderComponent(<AppLayout contentType="form" {...singleDrawer} />);
+    expect(wrapper.findDrawersTriggers()).toHaveLength(2);
+    expect(wrapper.findToolsToggle()).toBeFalsy();
+  });
+
+  test('renders drawers with the tools', () => {
+    const { wrapper } = renderComponent(<AppLayout contentType="form" {...singleDrawer} />);
```

Comment:
Do we need this test now with the one above?

Reply:
True, they test same use-case.

There will be merge conflicts with https://github.com/cloudscape-design/components/pull/1527. I will review these tests one more time when resolving the merge conflict